coursera/commandline.py

```diff
@@ -14,6 +14,14 @@
 from .utils import decode_input
 
 
+def class_required(args):
```

Comment:
Could you please document the function? Specifically, description, arguments and return value in epytext (similar to sphinx) doc string style, there are plenty of examples in the code.